eng/swagger_to_sdk_config.json

```diff
@@ -2,7 +2,7 @@
   "$schema": "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/documentation/sdkautomation/SwaggerToSdkConfigSchema.json",
   "generateOptions": {
     "generateScript": {
-      "path": "pwsh ./eng/scripts/Automation-Sdk-Generate.ps1"
+      "path": "pwsh ./eng/scripts/Invoke-GeneratAndBuildV2.ps1"
```

Comment:
What is the plan for Automation-Sdk-Generate.ps1? If we are planning to use it any longer we should delete it.

Reply:
We will not use Automation-sdk-Generate.ps1 anymore. I will delete it.